.github/CONTRIBUTING.md

```diff
@@ -14,6 +14,63 @@ Our SDKs are open-source and we do accept pull-requests if you feel like taking
 
 We require pull-requests for code and documentation to be submitted against the `master` branch in order to review and run it in our gated build system. We try to maintain a high bar for code quality and maintainability, we insist on having tests associated with the code, and if necessary, additions/modifications to the requirement documents.
 
+#### Contributing guidelines:
```

Comment:
Add a line that says , 

Please follow the below contribution guidelines before submitting a PR. (with the link to contribution GL)